src/native/src/core/nova_renderer.h

```diff
@@ -7,12 +7,13 @@
 
 #include <memory>
 
-#include "../interfaces/iwindow.h"
+#include "interfaces/iwindow.h"
+#include "mc/mc_objects.h"
+
 #include "nova.h"
-#include "../mc/mc_objects.h"
+#include "types.h"
```

Comment:
Nope. types.h is for typedefs that make the code more readable. mc_objects.h is for C-style structs that the Nova DLL will export. Don't put mc_objects.h in types.h.